<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcWidget</class>
 <widget class="QWidget" name="CalcWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Crypto/fiat and fiat/fiat calculator.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_configure">
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>18</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineFrom">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>From...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboCalcFrom"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="DoublePixmapLabel" name="imageExchange">
       <property name="text">
        <string>exchange image</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineTo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>To...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboCalcTo"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="text">
      <string>Exchange rates are updated every 2 minutes.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoublePixmapLabel</class>
   <extends>QLabel</extends>
   <header>components.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>fromChanged(QString)</slot>
  <slot>toChanged(QString)</slot>
  <slot>toComboChanged(QString)</slot>
 </slots>
</ui>
